providers/aws/cognito.go

```diff
@@ -0,0 +1,88 @@
+package aws
+
+import (
+	"context"
+	"log"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cognitoidentity"
+	"github.com/aws/aws-sdk-go-v2/service/cognitoidentityprovider"
+)
+
+type CognitoGenerator struct {
+	AWSService
+}
+
+func (g *CognitoGenerator) loadIdentityPools(svc *cognitoidentity.Client) error {
+	// TODO:
+	// * Cognito does not provides a `NewIdentityPoolsPaginator` like other services' "NewListClustersPaginator"
+	// ? Replace if it being avaialble, 60 being the max under constraint currently
+	const maxIdentityPool = 60
```

Comment:
~/Codes/terraformer/vendor/github.com/aws 

❯ ag 'MaxResults \*int'  | grep 'required'
```
aws-sdk-go-v2/service/cognitoidentity/api_op_ListIdentityPools.go:19:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/cognitoidentity/api_op_ListIdentities.go:29:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/cognitoidentityprovider/api_op_ListUserPools.go:20:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/cognitoidentityprovider/api_op_ListUserImportJobs.go:19:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
aws-sdk-go-v2/service/ec2/api_op_DescribeByoipCidrs.go:25:	MaxResults *int64 `min:"1" type:"integer" required:"true"`
```
❯ ag 'MaxResults \*int'  | grep -v 'required'
(most other services)

This param's definition does differ with other service's, so as the behavior. (e.g. leaving out MaxResults in ListUserPoolClientsInput would make the call refuses to return results, while leaving out MaxResults for other ListXXXXXXXInput would have a default value at aws's side). 

So I guess it might need to go towards the paginator way to support this one fully, could be separate changes/PR in sharable files.